packages/core-components/src/components/Table/Table.tsx

```diff
@@ -365,6 +365,15 @@ export function Table<T extends object = {}>(props: TableProps<T>) {
     setSelectedFiltersLength(selectedFiltersArray.flat().length);
   }, [data, selectedFilters, getFieldByTitle]);
 
+  // Check for deprecated checkbox-tree filter
+  useEffect(() => {
+    filters?.map(filter => {
```

Comment:
could also do `if (filters?.some(filter => filter.type === 'checkbox-tree')) { console.warn..`

Reply:
I did too much copy+paste, not enough reading! Thanks :)